website/templates/public/pages/help/addons.mako

```diff
@@ -1,10 +1,10 @@
 <p>The OSF makes integrating your various research tools easy. We are always developing Add-On connections to services
     you already use. An add-on is a connection between the OSF to another, existing tool.</p>
```

Comment:
I know this paragraph isn't a part of your edits, but there is a lot of unnecessary text here. I believe we can remove the 2nd sentence here; it is not useful information for a user.